test/LibraryManager.IntegrationTest/.editorconfig

```diff
@@ -0,0 +1,4 @@
+# Dotnet code style settings:
```

Comment:
What's the intent of adding this .editorconfig?  Style rules should go into the root .editorconfig so that we have consistent style throughout the project, but you're overriding it here.

Reply:
This would be created by default when creating new solution here. I can delete that as unnecessary.